app-tracking-protection/vpn-api/src/main/java/com/duckduckgo/mobile/android/vpn/feature/AppTpSetting.kt

```diff
@@ -20,11 +20,11 @@ enum class AppTpSetting(override val value: String, override val defaultValue: B
     BadHealthMitigation("badHealthMitigation", defaultValue = true),
     Ipv6Support("ipv6Support"),
     PrivateDnsSupport("privateDnsSupport"),
-    NetworkSwitchHandling("networkSwitchSupport"),
-    SetActiveNetworkDns("setActiveNetworkDns"),
+    NetworkSwitchHandling("handleNetworkSwitching", defaultValue = true),
```

Comment:
`networkSwitchSupport` -> `handleNetworkSwitching` to ensure that older clients won't be affected. As with the new networking layer (and legacy) this will be on by default